Index: gobject-introspection-1.34.2/Makefile.am
===================================================================
--- gobject-introspection-1.34.2.orig/Makefile.am	2012-11-12 20:46:15.000000000 +0000
+++ gobject-introspection-1.34.2/Makefile.am	2013-03-28 11:24:23.660940626 +0000
@@ -12,20 +12,19 @@
 GTESTER_PROGS =
 noinst_PROGRAMS = $(GTESTER_PROGS)
 
-include common.mk
 include Makefile.introspection
 
-include Makefile-cmph.am
-include Makefile-girepository.am
-include Makefile-giscanner.am
-include Makefile-examples.am
+#include Makefile-cmph.am
+#include Makefile-girepository.am
+#include Makefile-giscanner.am
+#include Makefile-examples.am
 include Makefile-gir.am
-include Makefile-tools.am
+#include Makefile-tools.am
 
 ## Process this file with automake to produce Makefile.in
 ACLOCAL_AMFLAGS = -I m4 ${ACLOCAL_FLAGS}
 
-SUBDIRS = . docs tests
+SUBDIRS = .
 
 DIST_SUBDIRS = m4 $(SUBDIRS)
 DISTCHECK_CONFIGURE_FLAGS = --enable-gtk-doc
