Index: gobject-introspection-1.34.2/giscanner/gdumpparser.py
===================================================================
--- gobject-introspection-1.34.2.orig/giscanner/gdumpparser.py	2012-03-08 14:16:01.000000000 +0000
+++ gobject-introspection-1.34.2/giscanner/gdumpparser.py	2013-03-28 11:10:23.751519032 +0000
@@ -158,6 +158,7 @@
         out_path = os.path.join(self._binary.tmpdir, 'dump.xml')

         args = []
+        args.append('g-ir-runme')
         args.extend(self._binary.args)
         args.append('--introspect-dump=%s,%s' % (in_path, out_path))

